charts/trento-server/Chart.yaml

```diff
@@ -7,7 +7,7 @@ description: The trento server chart contains all the components necessary to ru
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 1.1.1-dev
+version: 1.1.1-dev1
```

Comment:
On a second thought, is this semver? by reading the spec I *think* this should be `1.1.1-dev.1`>
@stefanotorresi HALP :-)

Reply:
Yes, you are right. I'll change it

Reply:
Done